mlrun/serving/server.py

```diff
@@ -112,6 +113,7 @@ def add_model(
         class_args = deepcopy(class_args)
         class_args["model_path"] = model_path
         route = ServingTaskState(class_name, class_args, handler)
+        namespace = namespace or inspect.stack()[1][0].f_globals
```

Comment:
what is `[1][0]` ? do they always exists ? or there is a scenario where this would raise Index out of range ?
I would suggest to have some helper function `get_X_globals` will help understand what is actually returned from `inspect.stack()[1][0].f_globals` and also there you will be able to do "safe" navigation on `inspect.stack()`

Reply:
done, added `get_caller_globals()`